src/splitwisesdk/core/Activity.java

```diff
@@ -0,0 +1,15 @@
+package splitwisesdk.core;
+
+public class Activity {
+    public boolean isPositive; // doubt
+    public String message;
+    public String category;
+    public String subMessage;
+    public String date;
+
```

Comment:
Add with its getter and setter
private String activityType;

Also add following static members
private static String EXPENSE_ACTIVITY = "EXPENSE_ACTIVITY";
private static String SETTLEMENT_ACTIVITY = "SETTLEMENT_ACTIVITY";